mcs/class/corlib/ReferenceSources/Buffer.cs

```diff
@@ -11,6 +11,9 @@ namespace System
 {
 	partial class Buffer
 	{
+		[MethodImplAttribute(MethodImplOptions.InternalCall)]
+		internal static extern unsafe bool InternalMemcpy (byte *dest, byte *src, int count);
```

Comment:
`ves_icall_System_Buffer_MemcpyInternal` is void, why is the return type declared as bool here?